### ISSUE TITLE
Starting Atom from command line on windows: "xx.TMP not found"

### ISSUE THREAD

**Issue by ccoenen** [atom/atom#11239]
I am running *Atom 1.6.0 on Windows 8.1* . When I start atom from the command line (say, in a directory using `atom .` or just plain `atom`), the command line will eventually start atom, but it will also print a line like:

```
C:\Users\test\projects\projectA>atom .
Datei ,92.TMP nicht gefunden
,92.tmpC:\Users\test\projects\projectA\92.tmp konnte nicht gefunden werden

C:\Users\test\projects\projectA>
```
(missing newline in line 4 is exactly as it appears in cmd.exe, it's not a copy&paste oversight).

The Error message translates (loosely?) to:

```
C:\Users\test\projects\projectA>atom .
File ,92.TMP not found
,92.tmpC:\Users\test\projects\projectA\92.tmp could not be found

C:\Users\test\projects\projectA>
```
The number (92 in this case) will vary each time, the rest is constant. It happens every single invocation.

### Steps to reproduce:

* have a windows machine or VM
* have atom 1.6
* start a `cmd.exe` or Powershell
* start atom from that shell with `atom` or `atom .`

### Actual Behaviour

See above, you will get an error message

### Desired Behaviour

No error message

This is very recent behaviour, this did not happen to me, up until yesterday, it might be a regression in 1.6, I'm not sure.

**Comment by Ben3eeE** [comment]
Same issue running atom from cmd.exe in windows 7.

There was https://github.com/atom/atom/pull/10950/commits/44fb9a4521e7f05cfb80fcd7057fdf75a9cd57f0 that was closed fixing problems with getting PID. Adding the quotes around %T% as this commit does removes the error message for me.

cc @damieng @joefitzgerald

**Comment by joefitzgerald** [comment]
This is fixed in atom beta (https://atom.io/beta) so will hopefully make its way to you relatively soon. @lee-dohm we should think about pulling the atom.cmd fixes into the 1.6 branch.

**Comment by Ben3eeE** [comment]
The beta currently has an open issue with the atom.cmd/atom.sh scripts that maybe should be considered before pulling the fixes into the 1.6 branch.
See https://github.com/atom/atom/issues/11164#issuecomment-199409185

**Comment by ccoenen** [comment]
I can confirm that this issue is gone in 1.6.1. Thanks everyone!
